tests/cnf/core/network/metallb/internal/tsparams/mlbvars.go

```diff
@@ -85,6 +85,13 @@ var (
 	IBPGPProtocol = "iBGP"
 	// TestLabel represents node label for testing.
 	TestLabel = map[string]string{"test": "label"}
+	// IPForwardAndSleepCmd enables IPv4 forwarding and keep the container running command.
+	IPForwardAndSleepCmd = []string{
+		"/bin/bash",
+		"-c",
+		"echo 1 > /proc/sys/net/ipv4/ip_forward 2>/dev/null || true; " +
+			"sleep infinity",
```

Comment:
general note, sleep inf is bad because it does not listen for signals and that has some weird behavior.

Is recommended 
```
/bin/bash -c "trap : TERM INT; sleep infinity & wait"
```

Reply:
added